index.html

```diff
@@ -26,6 +26,8 @@
             SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
         -->
         <title>ECMAScript 6: New Features: Overview and Comparison</title>
+        <link rel="icon" type="image/png" href="https://res.cloudinary.com/codeandpush/image/upload/c_scale,w_42/v1488836532/logo_es6_qncxsf.png"/>
+        <link rel="shortcut icon" type="image/png" href="https://res.cloudinary.com/codeandpush/image/upload/c_scale,w_42/v1488836532/logo_es6_qncxsf.png"/>
```

Comment:
`shortcut` keyword is not be required, and ignored by modern browsers. See https://stackoverflow.com/a/13211460/1571491